go/libkb/saltpack_enc.go

```diff
@@ -18,8 +17,13 @@ func SaltPackEncrypt(
 	for _, k := range receivers {
 		receiverBoxKeys = append(receiverBoxKeys, naclBoxPublicKey(k))
 	}
-	plainsink, err := saltpack.NewEncryptArmor62Stream(
-		sink, naclBoxSecretKey(sender), receiverBoxKeys)
+
+	var bsk saltpack.BoxSecretKey
+	if !sender.IsNil() {
```

Comment:
I think we conflate the two in some other contexts. Seems okay, to me, though, since the probability of generating an all-0 key is small, right?